### DISCUSSION TITLE
how to change legend position in chat

### DISCUSSION THREAD

**Discussion by Venkat05s** [vikramlearning/blazorbootstrap#481] [Q&A]
![Capture](https://github.com/vikramlearning/blazorbootstrap/assets/117189483/94f16a65-7be7-4b86-be35-140ebee5bae3)

**Comment by gvreddy04** [comment]
@Venkat05s Thanks for trying **BlazorBootstrap**! To set the legend position, you can use the following options: **left**, **right**, **top**, or **bottom**.

To change the legend position to the right, please update the code as follows:

```cs
protected override void OnInitialized()
{
    // Other implementation goes here

    pieChartOptions = new();
    pieChartOptions.Plugins.Legend.Position = "right";

    // Other implementation goes here
}
```

![image](https://github.com/vikramlearning/blazorbootstrap/assets/2337067/b9742835-a7a7-4d5c-9dff-62db1be2752e)
